### DISCUSSION TITLE
Blur QInput on keyup.enter without reference?

### DISCUSSION THREAD

**Discussion by dilly-sixmo** [quasarframework/quasar#15011] [General - Components / Directives / etc] [answered]
Is it possible to call a component's method (e.g, `QInput`'s `blur()` method) without creating a reference to the component if I'm calling that method from the component itself?

For example, I want to call the `blur()` method of a `QInput` when I press the Enter key while it has focus.  I'd like to do something like:
`<q-input v-model="price" @keyup.enter="blur">`

This obviously doesn't work.  Is the only solution to use a reference variable even if I'm calling a method from the component it belongs to?  Or is there some way to avoid that?  Something like a `this` keyword perhaps?

**Comment by metalsadman** [comment, marked as answer]
`@keyup.enter="(evt) => { evt.target.blur() }"`

**Reply by dilly-sixmo** [reply]
Yeah, that'll probably do it.  I guess sometimes when I code for so long the simplest things escape me.
